src/components/shared/Login.jsx

```diff
@@ -0,0 +1,56 @@
+import { GoogleLogin, GoogleLogout } from "react-google-login";
+import { useState, useEffect } from "react";
+
+const clientId =
```

Comment:
Keep it in env file

Reply:
.env file created and placed clientId there